src/server/pfs/fuse/filesystem.go

```diff
@@ -606,6 +612,14 @@ func (d *directory) lookUpFile(ctx context.Context, name string) (fs.Node, error
 	// path currently being looked up
 	directory := d.copy()
 	directory.File.Path = fileInfo.File.Path
+
+	// OBSOLETE: We need to remove the leading slash because of old error handling in pfs api/internalAPI layers
+	// This can go away once we collaps those layers
```

Comment:
This is true but we should figure out if `/foo` and `foo` are the same file or different.